### PR TITLE
[Feature] Surprise me [Closed]

### PR DESCRIPTION
**Pull request by plusminuschirag**
**Issue #40 Resolved.**

Say Surprise,Bored for Groot to listen and it will fire a link of video,article or entertaining site for you.

### CODE REVIEW

File: faceUnlocker.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import cv2
```
Comment (omi10859) [inline faceUnlocker.py]: this file should be in face unlock branch/pull request, Why changes are here?

File: notesWindow.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+from PyQt5.QtWidgets import QApplication,QWidget,QLabel,QTextEdit,QVBoxLayout,QGridLayout,QPushButton,QLineEdit,QSizePolicy,QScrollArea,QMessageBox
```
Comment (omi10859) [inline notesWindow.py]: this is a part of the note making feature. It should not be here i guess?

**Review by omi10859 — CHANGES_REQUESTED** [review]

File: notesWindow.py (hunk 2 above)
Comment (plusminuschirag) [reply notesWindow.py]: I didn't create branches from master. I had create them one after another. That' why I think this has happened.

Comment (plusminuschirag) [reply notesWindow.py]: I created them in order. That's why they are like this.

File: faceUnlocker.py (hunk 1 above)
Comment (plusminuschirag) [reply faceUnlocker.py]: I did them in Serialized way.

**Comment by omi10859** [comment]
Already told you to commit the changes with a proper commit message.
> "Cleaning Branch 3"

What will someone understand with this?

**Pull request closed without merging**